package.json

```diff
@@ -63,7 +63,7 @@
     "express": "^4.17.1",
     "filenamify": "^4.3.0",
     "fluent-ffmpeg": "^2.1.2",
-    "got": "^11.8.1",
+    "got": "^12.0.0",
```

Comment:
got v12 is now a pure ESM package. freyr can no longer require it.

https://gist.github.com/sindresorhus/a39789f98801d908bbc7ff3ecc99d99c

Reply:
freyr now supports esm. https://github.com/miraclx/freyr-js/pull/202